docs/editor/profiles.md

```diff
@@ -0,0 +1,169 @@
+
+Once you choose whether to create a new profile based on the current profile or an empty profile, you are prompted to enter a name for the new profile.
+
+### Check the current profile
+
+The current profile name is displayed in several places in the VS Code UI:
+
+* Title bar
+* **File** > **Preferences** > **Profiles**
+* **Manage** gear button hover
+
+If you are still using the Default Profile, no profile name is displayed.
+
+The **Manage** gear button displays a badge with the first two letters of the active profile so you can quickly check which profile you are running.
+
+![Manage gear displaying "DE' to indicate that the user's 'Demo' profile is active](images/profiles/profile-gear-two-letters.png)
+
+### Edit a profile
+
+You can edit a profile just as you would normally change any VS Code configuration. You can install/uninstall/disable extensions, change settings, and adjust snippets like normal. These changes are stored in your currently active profile.
+
+### Workspace associations
+
+When you select create or select an existing profile, it is associated with the current workspace and whenever you open that folder, the workspace's profile is active. If you open another folder, the profile either goes back to the Default Profile if no profile is set, or switches to that folder's associated profile.
+
+Setting a workspace's profile back to the Default Profile clears the workspace's profile association.
+
+## Managing profiles
+
+### Switch profiles
+
+You can quickly switch between profiles with the **Profiles: Switch Profile** command in the Command Palette, which presents a dropdown listing your available profiles.
+
+### Rename a profile
+
+You can rename an existing profile via the **Rename** command in the Profiles menu.
+
+### Delete a profile
+
+You can delete a profile via the **Delete Profile** command. The **Delete Profile** drop down lets you select which profile(s) to delete.
+
+## Profile contents
+
+The **Profiles: Show Contents** command (available in the Command Palette or Profiles menus) brings up the Profiles view, where you can review the customizations for the profile.
+
+![Profile view displaying contents of a Doc Work profile](images/profiles/profile-show-contents.png)
+
+A profile can include:
+
+* Settings - In a profile-specific `settings.json` file.
+* Extensions - You can remove extensions from the profile by unchecking their entry in the Profiles view.
+* UI state - Currently open views and their positioning.
+* Keybindings - In a profile-specific `keybindings.json` file.
+* Snippets - In a profile-specific `{language}.json` files.
+* User Tasks - In a profile-specific `tasks.json` file.
+
+When you create a new profile based on the Default Profile, the profile-specific configuration files are populated from your user configuration files. Workspace-specific settings are not automatically included in a new profile.
```

Comment:
Here we should mention what happens if you do not want something to be a part of a profile. In this case, we actually use the content from the default profile.
Example: if you import a profile, and uncheck the keybindnigs, the profile you import will instead use keybindings from the default profile.

Reply:
We can also mention this in the import flow. Whatever you think works better.

Reply:
Hmm, this is subtle and somewhat confusing behavior. I don't see this behavior for example with extensions. If I have an extension in a profile and I unselect the extension before exporting, when I import I don't see my Default Profile extensions.
Let's leave this out for now and we can iterate on the best way to describe this behavior.